common/src/main/java/bio/terra/catalog/service/CatalogStatusService.java

```diff
@@ -13,31 +13,31 @@
 import org.springframework.stereotype.Service;
 
 @Service
-public class CatalogStatusService extends BaseStatusService {
+public class CatalogStatusService {
   private static final Logger logger = LoggerFactory.getLogger(CatalogStatusService.class);
 
   private final NamedParameterJdbcTemplate jdbcTemplate;
 
   @Autowired
   public CatalogStatusService(
+      StatusCheckService statusCheckService,
       NamedParameterJdbcTemplate jdbcTemplate,
-      StatusCheckConfiguration configuration,
       SamService samService,
       DatarepoService datarepoService,
       RawlsService rawlsService) {
-    super(configuration);
     this.jdbcTemplate = jdbcTemplate;
-    registerStatusCheck("CloudSQL", this::databaseStatus);
-    registerStatusCheck("SAM", samService::status);
-    registerStatusCheck("Data Repo", datarepoService::status);
-    registerStatusCheck("Rawls", rawlsService::status);
+    statusCheckService.registerStatusCheck("CloudSQL", this::databaseStatus);
+    statusCheckService.registerStatusCheck("SAM", samService::status);
+    statusCheckService.registerStatusCheck("Data Repo", datarepoService::status);
+    statusCheckService.registerStatusCheck("Rawls", rawlsService::status);
   }
 
-  private SystemStatusSystems databaseStatus() {
+  @VisibleForTesting
+  SystemStatusSystems databaseStatus() {
     try {
       logger.debug("Checking database connection valid");
       return new SystemStatusSystems()
-          .ok(jdbcTemplate.getJdbcTemplate().execute((Connection conn) -> conn.isValid(5000)));
+          .ok(jdbcTemplate.getJdbcTemplate().execute((Connection conn) -> conn.isValid(1)));
```

Comment:
Thoughts on changing 1 to a constant, perhaps something named DELAY_SECONDS
Naming the number may make the intent and assumption of the developer clearer in the code for the next person.

Reply:
Sure, I can add a constant here to avoid the magic number.